### DISCUSSION TITLE
Unable to push code error 403

### DISCUSSION THREAD

**Discussion by VishwaPratapSingh21** [community/community#38863] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

remote: Permission to VishwaPratapSingh21/Chat_App.git denied to roy-sub.
fatal: unable to access 'https://github.com/VishwaPratapSingh21/Chat_App.git/': The requested URL returned error: 403
I am getting this error while pushing my code
please help me

**Comment by MathiAs2Pique** [comment]
Hello,

What are you using to push the code ? Github Desktop, cli ? On which platform ?

**Comment by DSDmark** [comment]
Hi @VishwaPratapSingh21,

> if you are using 2FA with GitHub, you cannot use the actual password from remote, you need to create an Access Token and use that instead of your password.

here how to generate token:- [Creating a personal access token](https://docs.github.com/en/enterprise-server@3.4/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)

generate token from here:- https://github.com/settings/tokens
